workloads/burn-in/README.md

```diff
@@ -6,15 +6,21 @@ This repository contains scripts and configuration files to use the GPU optimize
 
 If the expected performance is seen, the user can be confident that the nodes are working correctly.  If a Top500 submittal is desired, there are additional optimizations than can done to maximize performance.  Please work with an NVIDIA Solutions Architect (SA) to review the Burn In Test results so that additional perfomance may be obtained.
 
-Currently, only clusters built with DGX-1V-16GB, DGX-1V-32GB, and DGX2 are supported.  If you have an OEM GPU-based system, contact your NVIDIA SA for additional assistance.
+Currently, only clusters built with DGX-1V-16GB, DGX-1V-32GB, DGX-2, and DGXA100 are supported.  If you have an OEM GPU-based system, contact your NVIDIA SA for additional assistance.
 
 ## Requirements
 
 - This repository does not include the HPL binaries.  Contact your NVIDIA Solutions Architect or other NVIDIA representative for access.
 
 ## Getting started
 
-First copy the hpl-burnin test directory to the shared home directory of the slurm cluster login node. Then place the hpl binary in the hpl-burnin directory and run launch_experiment_slurm.sh from it without any options.
+From the DeepOps repo, either copy the entire DeepOps repo or just the hpl-burnin test directory to the user's home directory of the slurm cluster to be tested. It is assumed that this directory is on a shared filed system. Place the hpl binary in the hpl-burnin directory (`deepops/tests/hpl-burnin`) and run launch_experiment_slurm.sh from it without any options.
```

Comment:
This needs to be updated to the new path.

I also think we should just tell them to clone/copy the whole DeepOps directory rather than giving them the option to just use the hpl-directory.

Reply:
Yes we should.  I think the reason here is that the process when downloading the tool or actually using deepops should be the same.  On a deepops system, the deepops directory is not on a share system and it was thought to copy the hpl-burnin directory to the shared directory.  I have no preference here.

Reply:
I will update the path.  Craig is correct.  The assumed flow is that you have deepops downloaded to a separate machine that is used to setup the cluster (login and compute nodes, nfs).  Then if you wanted to run burn in test, you would just need to copy the burn-in directory to that cluster because all you would need is that folder to run the test.